pkg/validation/limits.go

```diff
@@ -107,6 +109,10 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.MaxLocalStreamsPerUser, "ingester.max-streams-per-user", 10e3, "Maximum number of active streams per user, per ingester. 0 to disable.")
 	f.IntVar(&l.MaxGlobalStreamsPerUser, "ingester.max-global-streams-per-user", 0, "Maximum number of active streams per user, across the cluster. 0 to disable.")
 	f.BoolVar(&l.UnorderedWrites, "ingester.unordered-writes", false, "(Experimental) Allow out of order writes.")
+	_ = l.MaxLocalStreamRateBytes.Set("10000")
+	f.Var(&l.MaxLocalStreamRateBytes, "ingester.max-stream-rate-bytes", "Maximum bytes per second rate per active stream.")
+	_ = l.MaxLocalStreamBurstRateBytes.Set("20000")
+	f.Var(&l.MaxLocalStreamBurstRateBytes, "ingester.max-stream-rate-bytes", "Maximum burst bytes per second rate per active stream.")
```

Comment:
Please mimic how we set similar configs:
https://github.com/grafana/loki/blob/main/pkg/ingester/wal.go#L24-L50

Notably:
1) Use a readable `const default ${VARIABLE}` with appropriate comment if applicable
2) Default to a powers of 2 form. 

I'd like us to default to the following
```
const defaultPerStreamRateLimit = 1<<20 // 1MB
const defaultPerStreamBurstLimit = 2 * defaultPerStreamRateLimit
```